organisations/relationship_transfer.go

```diff
@@ -0,0 +1,115 @@
+package organisations
+
+import (
+	"fmt"
+	"github.com/Financial-Times/neo-utils-go/neoutils"
+	"github.com/jmcvetta/neoism"
+)
+
+type relationships []struct {
+	RelationshipType string `json:"relationship"`
+}
+
+// TransferRelationships is responsible for moving relationships from node with sourceUUID to node with destinationUUID
+func TransferRelationships(cypherRunner neoutils.CypherRunner, destinationUUID string, sourceUUID string) ([]*neoism.CypherQuery, error) {
```

Comment:
Picky but this function is only creating the query and in other places you have made it clear in the name of the method, should we do the same here?